src/test/suite/text-register.test.ts

```diff
@@ -33,7 +33,7 @@ suite("Text registers", () => {
     await emulator.runCommand("forwardChar");
```

Comment:
Revert all these wrong changes.
The current impl of these commands don't have to be modified.